modules/mcg-about-multicloud-gitops.adoc

```diff
@@ -0,0 +1,70 @@
+:_content-type: CONCEPT
+:imagesdir: ../../images
+
+[id="about-multicloud-gitops-pattern"]
+= About the Multicloud GitOps pattern
+//AI:needs SME review
+
+//AI self: Note that Description list does not render correctly on the site
+Use case::
+
+* Use a GitOps approach to manage hybrid and multi-cloud deployments across both public and private clouds.
+* Enable cross-cluster governance and application lifecycle management.
+* Securely manage secrets across the deployment.
++
+[NOTE]
+====
+Based on the requirements of a specific implementation, certain details might differ. However, all validated patterns that are based on a portfolio architecture, generalize one or more successful deployments of a use case.
+====
+
+Background::
+Organizations are aiming to develop, deploy, and operate applications on an open hybrid cloud in a stable, simple, and secure way. This hybrid strategy includes multi-cloud deployments where workloads might be running on multiple clusters and on multiple clouds- private or public.
+This strategy requires an infrastructure-as-code approach: GitOps. GitOps uses Git repositories as a single source of truth to deliver infrastructure-as-code. Submitted code checks the continuous integration (CI) process, while the continuous delivery (CD) process checks and applies requirements for things like security, infrastructure-as-code, or any other boundaries set for the application framework. All changes to code are tracked, making updates easy while also providing version control should a rollback be needed.
+
+[id="about-solution"]
+== About the solution
+
+This architecture covers hybrid and multi-cloud management with GitOps as shown in following figure. At a high level this requires a management hub, for DevOps and GitOps, and infrastructure that extends to one or more managed clusters running on private or public clouds. The automated infrastructure-as-code approach can manage the versioning of components and deploy according to the infrastructure-as-code configuration.
+
+Benefits of Hybrid Multicloud management with GitOps:
+
+* Unify management across cloud environments.
+* Dynamic infrastructure security.
+* Infrastructural continuous delivery best practices.
+
+//figure 1 originally
+.Overview of the solution including the business drivers, management hub, and the clusters under management
+image::multicloud-gitops/hybrid-multicloud-management-gitops-hl-arch.png[Multicloud Architecture]
+
+//AI: Don't think we need this, yet get it confirmed
+//[discrete]
+//[id="logical-diagram"]
+//== Logical diagram
+
+In the following figure, logically, this solution can be viewed as being composed of an automation component, unified management (including secrets management), and the clusters under management- all running on top of a user-chosen mixture of on-premise data centers and public clouds.
```

Comment:
- Remove parentheses.
- "and the clusters under management-" remove the dash, maybe replace with a comma.